tagstudio/src/core/media_types.py

```diff
@@ -392,6 +392,11 @@ class MediaCategories:
         extensions=_IMAGE_VECTOR_SET,
         is_iana=False,
     )
+    IMAGE_RASTER_TYPES: MediaCategory = MediaCategory(
+        media_type=MediaType.IMAGE,
+        extensions=_IMAGE_SET,
```

Comment:
If `IMAGE_RASTER_TYPES` is going to pull from the same `IMAGE_SET`, then that set should probably be renamed to `_IMAGE_RASTER_SET` to avoid future confusion